src/common/Graylog.cc

```diff
@@ -85,7 +85,7 @@ void Graylog::log_entry(Entry const * const e)
     m_formatter->dump_string("short_message", s);
     m_formatter->dump_string("_app", "ceph");
     m_formatter->dump_float("timestamp", e->m_stamp.sec() + (e->m_stamp.usec() / 1000000.0));
-    m_formatter->dump_int("_thread", e->m_thread);
+    m_formatter->dump_int("_thread", (int64_t)e->m_thread);
```

Comment:
seems like this should really be dump_unsigned and uint64_t, since it's a pointer